presto-native-execution/src/test/java/com/facebook/presto/nativeworker/TestPrestoNativeIcebergTpcdsWithPositionalDeletes.java

```diff
@@ -0,0 +1,265 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.nativeworker;
+
+import com.facebook.presto.Session;
+import com.google.common.collect.ImmutableMap;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+
+@Test(groups = {"parquet"})
+public class TestPrestoNativeIcebergTpcdsWithPositionalDeletes
+        extends TestPrestoNativeIcebergTpcdsQueriesParquetUsingThrift
+{
+    Map<String, Long> deletedRowsMap;
+    private void doDeletes()
+    {
+        ImmutableMap.Builder<String, Long> deletedRowsMapBuilder = ImmutableMap.builder();
+
+        assertUpdateExpected(session, "DELETE FROM inventory " +
```

Comment:
Move the inventry delete to the end to match the delete function order in the spec